db/migrate/20230327024908_create_developer_external_profiles.rb

```diff
@@ -0,0 +1,13 @@
+class CreateDeveloperExternalProfiles < ActiveRecord::Migration[7.0]
+  def change
+    create_table :developer_external_profiles do |t|
+      t.references :developer, null: false, foreign_key: true
+      t.string :site, null: false
+      t.jsonb :data
```

Comment:
```suggestion
      t.jsonb :data, null: false, default: {}
```

That way we always can access the hash directly, instead of checking if it exists first.